pybleau/app/io/tests/test_analyzer_roundtrip.py

```diff
@@ -100,21 +106,21 @@ def base_round_trip_analysis_and_plotter_with_plot(self, style, config_klass, an
         return new_analysis
 
     def assert_roundtrip_identical(self, obj, **kwargs):
-        SKIPS = {"source_analyzer", "canvas_manager", "uuid", "plot",
-                 "inspectors", "plot_factory"}
+        skips = ANALYSIS_SKIPS | PLOTTER_SKIPS
         return assert_roundtrip_identical(obj, serial_func=serialize,
                                           deserial_func=deserialize,
-                                          ignore=SKIPS, **kwargs)
+                                          ignore=skips, **kwargs)
 
 
-class TestRoundTripMultiDfAnalyzer(TestCase, BaseAnalysisRoundTrip):
+class TestRoundTripBareAnalyzer(TestCase, BaseAnalysisRoundTrip):
```

Comment:
The name of this class changed, but it is still using a `MultiDataFramAnalyzer`. This is intended?

Reply:
Good catch, reverting...